index.js

```diff
@@ -48,19 +48,19 @@ async function fastifyStatic (fastify, opts) {
     maxAge: opts.maxAge
   }
 
-  const allowedPath = opts.allowedPath
-
-  if (opts.prefix === undefined) opts.prefix = '/'
-
   let prefix = opts.prefix
+  if (prefix === undefined) {
+    prefix = (opts.prefix = '/')
+  }
```

Comment:
thought you would reduce it to this :D

```suggestion
  let prefix = opts.prefix ?? (opts.prefix = '/')
```

Reply:
Yeah I should maybe :D

`??` accepts `null` as well, so I didn't want to change semantics

Reply:
prefix has to be a string imho. so null should be also have the same behaviour.